ogr/parsing.py

```diff
@@ -136,16 +136,21 @@ def parse_git_repo(potential_url: str) -> Optional[RepoUrl]:
         )
     if not username and len(split) >= 2:
         # path contains username/reponame
+        # or some/namespace/reponame
+        # or fork/username/some/namespace/reponame
 
-        is_fork = "fork" in split
-        if is_fork and len(split) == 4:
+        is_pagure_fork = "fork" in split
+        if is_pagure_fork and len(split) == 4:
```

Comment:
I wonder about https://gitlab.com/fork, since you can nest groups on gitlab...

Reply:
The current version checks only the first part to `fork` and https://gitlab.com/fork has not subgroups yet so it's probably safe.